.gitignore

```diff
@@ -12,3 +12,4 @@ include/*
 share/*
 .tox/*
 pip-selfcheck.json
+.vscode/
```

Comment:
It's usually a good idea to not to update repositories ignore files' to include your personal editor's -- there're just a ton of editors out there.  This might be helpful for you in the future, though: for adding globs like this to the list of paths to ignore for a repository, your local git clone has a folder in `.git/info/exclude` that you can add globs to that *won't* be included as part of your commit or push.  You can find more information about that [here](https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-files-locally-only).

That being said, since I just happen to have at long last switched from vim to vscode, I'm not too worried about this. ;-)